Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -114,13 +114,13 @@ std::unique_ptr<optparse::OptionParser> CreateParser(ParserOptions options)
 
 static void AddConfigLayer(const optparse::Values& options)
 {
+  std::list<std::string> config_args;
   if (options.is_set_by_user("config"))
-  {
-    const std::list<std::string>& config_args = options.all("config");
-    Config::AddLayer(std::make_unique<CommandLineConfigLayerLoader>(
-        config_args, static_cast<const char*>(options.get("video_backend")),
-        static_cast<const char*>(options.get("audio_emulation"))));
-  }
+    config_args = options.all("config");
+
+  Config::AddLayer(std::make_unique<CommandLineConfigLayerLoader>(
+      config_args, static_cast<const char*>(options.get("video_backend")),
```

Comment:
`std::move(config_args)` - even **if** it doesn't result in a move now, it's worth futureproofing like this because we may change `AddLayer` to support moves in the future.